doc/radosgw/multisite.rst

```diff
@@ -751,15 +751,24 @@ to a multi-site system, follow these steps:
       radosgw-admin zonegroup rename --rgw-zonegroup default --zonegroup-new-name=<name>
       radosgw-admin zone rename --rgw-zone default --zone-new-name us-east-1 --rgw-zonegroup=<name>
 
-3. Configure the master zonegroup. Replace ``<name>`` with the realm name or
+3. To set the default zonegroup, replace the JSON file's ``api_name`` field with
+   the zonegroup name, and run the following command:
+
+   .. prompt:: bash #
+
+      radosgw-admin zonegroup set --infile zonegroup.json
+
+   Where ``zonegroup.json`` is `the JSON file you created <#setting-a-zonegroup>`_.
```

Comment:
if this json-wrangling step is only used to change the zonegroup's `api_name`, it would be simpler to use `radosgw-admin zonegroup modify --api-name=` for that

Reply:
Ok, modify now.